Source/Core/Core/PowerPC/JitArm64/JitArm64Cache.cpp

```diff
@@ -69,9 +75,35 @@ void JitArm64BlockCache::WriteLinkBlock(const JitBlock::LinkData& source, const
 void JitArm64BlockCache::WriteDestroyBlock(const JitBlock& block)
 {
   // Only clear the entry points as we might still be within this block.
-  ARM64XEmitter emit(block.checkedEntry);
+  ARM64XEmitter emit(block.checkedEntry, block.normalEntry + 4);
```

Comment:
Ditto, but for the number 4.

Reply:
I don't think this one needs any explaining if you look at the loop condition below.